test/views/components/LatePaymentPenaltySummaryCardSpec.scala

```diff
@@ -194,13 +194,13 @@ class LatePaymentPenaltySummaryCardSpec extends SpecBase with ViewBehaviours {
     "given a LPP1" should {
       implicit val doc: Document = asDocument(summaryCardHtml.apply(summaryCardModel))
 
-      "display the penalty amount" in {
-        doc.select("h4").text() shouldBe "£400 penalty"
+      "display the penalty amount and the date the VAT was due" in {
+        doc.select("h4").text() shouldBe "£400 penalty for late payment of charge due on 1 February 2020"
```

Comment:
Can you add or adjust a test for to verify the visually hidden part "for late payment..." only

Reply:
Done ✅